index.src.html

```diff
@@ -839,6 +914,26 @@ <h4 algorithm id="collectfromcredentialstore-passwordcredential">
             [=current settings object=]'s [=environment settings object/origin=].
   </ol>
 
+  <h4 algorithm id="create-passwordcredential">
+    `PasswordCredential`'s `[[Create]](data)`
+  </h4>
+
+  <dfn for="PasswordCredential" method>\[[Create]](data)</dfn> is called with a
+  {{CredentialCreationData}} (|data|), and returns a {{PasswordCredential}} if one can be created,
+  or `null` otherwise:
+
+  <ol class="algorithm">
+    1.  Assert: |data|["password"] [=map/exists=].
+
+    2.  If |data|["password"] is an {{HTMLFormElement}}, return the result of executing the
+        {{PasswordCredential(form)}} constructor on |data|["password"].
+
+    3.  If |data|["password"] is a {{PasswordCredentialData}}, return the result of executing the
```

Comment:
I eventually want this to be able to generate passwords too, but I don't mind getting just the async constructor first.